src/main/java/executor/command/CommandConvert.java

```diff
@@ -34,7 +34,8 @@ public CommandConvert(String userInput) {
         this.use = ""; // whether to use "from code" or "to code" for fetching exchange rate from json
         this.to = getCurrencyConvertTo(userInput);
         this.description = "Command that converts the user input cash amount from"
-                + " one currency to another and prints it on the User Interface.";
+                + " one currency to another and prints it on the User Interface.\n"
+                + "FORMAT : convert <amount up to 2dp> /from <Base> /to <Req>";
```

Comment:
Doesn't need to be 2dp right? Instead of Base perhaps you should right base currency e.g. USD? So that it is clearer what they are inputting in bc some of the might type the whole country's name

Reply:
Yup, i have done the required changes. I did put it to 2 dp because money wise usually its 2 dp ? But yes the api is able to handle more than 2dp. So wouldn't crash. Thanks for pointing that out ;)

Reply:
Only for amountStr i have left the try catch block and i have explained them with some test screenshots